data/snippets.ts

```diff
@@ -530,6 +530,12 @@ const coding = [
     keyword: "sql-select",
     type: "template",
   },
+  {
+    id: nanoid(),
+    name: "Get Name of Current Branch",
+    text: "git branch --contains | cut -d \" \" -f 2",
+    keyword: "git-current-branch",
```

Comment:
missing `type: "template",`

```suggestion
    keyword: "git-current-branch",
    type: "template",    
```